resource_customizations/argoproj.io/EventBus/health.lua

```diff
@@ -0,0 +1,21 @@
+hs={ status = "Progressing", message = "Waiting for initialization" }
+
+if obj.status ~= nil then
+  if obj.status.conditions ~= nil then
+    for i, condition in ipairs(obj.status.conditions) do
```

Comment:
maybe use `_` instead of `i` since you don't use i? https://stackoverflow.com/questions/39991717/looping-over-array-values-in-lua